src/cartservice/Dockerfile

```diff
@@ -18,8 +14,20 @@ COPY . .
 ARG SKAFFOLD_GO_GCFLAGS
 RUN go build -gcflags="${SKAFFOLD_GO_GCFLAGS}" -o /go/src/hipstershop/cartservice .
 
-# Deployment container
-FROM scratch
+# # Deployment container
+FROM alpine:latest
+
+# Install dependencies
+RUN apk --update --no-cache add ca-certificates make protoc
+
+# These tools are for debuggin the containres
+RUN apk update && apk add --no-cache \
```

Comment:
is it worth it to rerun the update? wasn't it executed in the previous line? I believe that you could merge these two.

Reply:
good catch